cmd/fleetctl/testdata/expectedGetConfigAppConfigYaml.yml

```diff
@@ -26,6 +26,12 @@ spec:
       enable_disk_encryption: false
     macos_setup:
       bootstrap_package: ""
+    end_user_authentication:
+      idp_name: ""
+      issuer_uri: ""
+      metadata: ""
+      metadata_url: ""
```

Comment:
note that since we're sharing the underlying `SSOProviderSettings` struct, we get `metadata` here, even though is not specified in the issue (and not used in the UI)

my thinking is that it's OK, but I can build a workaround if anyone thinks this is problematic.